### PR TITLE
Add basic end to end test for Validation statistics

### PR DESCRIPTION
**Pull request by zhilingc**
**What this PR does / why we need it**:
End to end tests that just test that you are able to retrieve singular instances of statistics over a dataset or single day. Will add more complex tests when I have solidified how aggregation will work.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: sdk/python/feast/client.py [hunk 1]
```diff
@@ -755,7 +755,7 @@ def ingest(
         max_workers: int = max(CPU_COUNT - 1, 1),
         disable_progress_bar: bool = False,
         timeout: int = KAFKA_CHUNK_PRODUCTION_TIMEOUT,
-    ) -> None:
+    ) -> str:
```
[the remark this answers is not in the document]
Comment (zhilingc) [reply sdk/python/feast/client.py]: whoops. corrected.

File: tests/e2e/feature-validation.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+def test_basic_retrieval_by_single_dataset(client, dataset_basic):
+    stats = client.get_statistics(
+        feature_refs=["strings", "ints", "floats"],
+        store="bigquery",
+        dataset_ids=[dataset_basic["id"]],
+    )
+    expected_stats = statistics_pb2.DatasetFeatureStatisticsList()
+    with open("statistics/expected_output_basic_dataset.json", "r") as fo:
+        contents = fo.read()
+    json_format.Parse(contents, expected_stats)
```
[the remark this answers is not in the document]
Comment (zhilingc) [reply tests/e2e/feature-validation.py]: Great suggestion - done.

File: tests/e2e/feature-validation.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        features=[
+            Feature("strings", ValueType.STRING),
+            Feature("ints", ValueType.INT64),
+            Feature("floats", ValueType.FLOAT),
+        ],
+        entities=[Entity("entity_id", ValueType.INT64)],
+        max_age=Duration(seconds=100),
+    )
+    client.apply(fv_fs)
+    time.sleep(20)
```
[the remark this answers is not in the document]
Comment (zhilingc) [reply tests/e2e/feature-validation.py]: Good point, but maybe in a separate PR to the master branch? I'll create an issue about it

**Comment by zhilingc** [comment]
e2e tests should fail now.

File: tests/e2e/feature-validation.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ches) [reply tests/e2e/feature-validation.py]: 👍 Issue #524 for historical reference

**Pull request merged**